### PR TITLE
pacman*, pacman4console, yay, yaourt: add Korean translation

### PR DESCRIPTION
**Pull request by CodePsy-2001**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

----

**Note to other maintainers**: Our Korean reviewer (native speaker/code owner) has been inactive for a while, so I have validated all pages with Google translate, we can wait for a few more days for their review. If not reviewed by then, we can merge this PR without their review.

**Review by IMHOJEONG — APPROVED** [review]
Thank you for your contribution. LGTM!

**Pull request merged**
